src/test/java/teammates/sqllogic/core/FeedbackQuestionsLogicTest.java

```diff
@@ -85,42 +93,53 @@ public void testGetFeedbackQuestionsForSession_questionNumbersOutOfOrder_success
         assertTrue(questions.containsAll(actualQuestions));
     }
 
-    @Test(enabled = false)
+    @Test
     public void testCreateFeedbackQuestion_questionNumbersAreConsistent_canCreateFeedbackQuestion()
             throws InvalidParametersException {
-        FeedbackSession fs = typicalDataBundle.feedbackSessions.get("session1InCourse1");
-        FeedbackQuestion fq1 = typicalDataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
-        FeedbackQuestion fq2 = typicalDataBundle.feedbackQuestions.get("qn2InSession1InCourse1");
-        FeedbackQuestion fq3 = typicalDataBundle.feedbackQuestions.get("qn3InSession1InCourse1");
-        FeedbackQuestion fq4 = typicalDataBundle.feedbackQuestions.get("qn4InSession1InCourse1");
-        FeedbackQuestion fq5 = typicalDataBundle.feedbackQuestions.get("qn5InSession1InCourse1");
-
-        List<FeedbackQuestion> questionsBefore = List.of(fq1, fq2, fq3, fq4);
+        Course c = getTypicalCourse();
+        FeedbackSession fs = getTypicalFeedbackSessionForCourse(c);
+        FeedbackQuestion fq1 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq2 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq3 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq4 = getTypicalFeedbackQuestionForSession(fs);
+        FeedbackQuestion fq5 = getTypicalFeedbackQuestionForSession(fs);
+
+        fq1.setQuestionNumber(1);
+        fq2.setQuestionNumber(2);
+        fq3.setQuestionNumber(3);
+        fq4.setQuestionNumber(4);
+        fq5.setQuestionNumber(5);
+
+        ArrayList<FeedbackQuestion> questionsBefore = new ArrayList<>(List.of(fq1, fq2, fq3, fq4));
```

Comment:
And this, basically it seems repetitive

Reply:
added `createQuestionList`, and used it for creating questions with sequential question numbers 

 i was following the principles here: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html but perhaps overdone it in this case